javascript/src/index.js

```diff
@@ -0,0 +1,45 @@
+// XXX even though ethers is not used in the code below, it's very likely
+// it will be used by any DApp, so we are already including it here
+const { ethers } = require("ethers");
+
+const rollup_server = process.env.ROLLUP_HTTP_SERVER_URL;
+console.log("HTTP rollup_server url is " + rollup_server);
+
+async function handle_advance(data) {
+  console.log("Received advance request data " + JSON.stringify(data));
+  return "accept";
+}
+
+async function handle_inspect(data) {
+  console.log("Received inspect request data " + JSON.stringify(data));
+  return "accept";
+}
+
+var handlers = {
+  advance_state: handle_advance,
+  inspect_state: handle_inspect,
+};
+
+var finish = { status: "accept" };
+
+(async () => {
+  while (true) {
+    const finish_req = await fetch(rollup_server + "/finish", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ status: "accept" }),
+    });
+
+    console.log("Received finish status " + finish_req.status);
+
+    if (finish_req.status == 202) {
```

Comment:
```suggestion
    const HTTP_CODE_ACCEPTED = 202;
    if (finish_req.status == $HTTP_CODE_ACCEPTED) {
```

I don't like magic numbers, but I understand this code is just a template, won't block the PR for this.